packages/tizen_package_manager/example/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: tizen_package_manager_example
+description: Demonstrates how to use the tizen_package_manager plugin.
+publish_to: 'none' 
```

Comment:
There was no package which has version in example pubspec.yaml.
I referenced other package's yaml file.

Reply:
`package_info_plus` specifies it because it's checked in the integration test: https://github.com/flutter-tizen/plugins/blob/1becaddcf442da0d2a858f753d02e832b755eaca/packages/package_info_plus/example/pubspec.yaml#L3

https://github.com/flutter-tizen/plugins/blob/1becaddcf442da0d2a858f753d02e832b755eaca/packages/package_info_plus/example/integration_test/package_info_plus_test.dart#L18

The default value of `version` in `pubspec.yaml` is `0.0.0` if it's omitted: https://dart.dev/tools/pub/pubspec#version. For Tizen, however, the tool checked `tizen-manifest.xml` to create tpk version. My suggestion is to specify the version explicitly for clarity and consistency with `package_info_plus`.